chatgpt-shell.el

```diff
@@ -65,6 +65,15 @@
   :type 'function
   :group 'chatgpt-shell)
 
+(defcustom chatgpt-shell-default-prompts '("Write a unit test for the following code:"
```

Comment:
nit: I may tweak defaults later, but it can always be overriden by users.

Reply:
Yep, that was the idea.